src/Xml2Json.php

```diff
@@ -127,7 +127,7 @@ protected static function processXml($simpleXmlElementObject, $ignoreXmlAttribut
         $value      = static::getXmlValue($simpleXmlElementObject);
         $attributes = (array) $simpleXmlElementObject->attributes();
 
-        if (! count($children)) {
+        if (! $children) {
```

Comment:
@samsonasik This is invalid. `$children` is `SimpleXMLElement` and just `!` doesn't work with objects. We need to have `count` there.

Reply:
Hm... it looks like it works somehow for this object ! Just tested on 3v4l... I am surprised.

Need to know reason why. ❓

Reply:
Ok, @samsonasik I have got an answer:
https://www.php.net/manual/en/language.types.boolean.php#language.types.boolean.casting
> When converting to boolean, the following values are considered FALSE:
> - SimpleXML objects created from empty tags
> Every other value is considered TRUE (including any resource and NAN).

So the change is fine, but imho is confusing...

I think we should leave it or eventually change it to `->count()`.

What do you think?

Reply:
calling `->count()` meant we will call both `children()` (used in line 147) and `count()`, that means more function to be called, from 1 to 2.